animation_nodes/nodes/matrix/distribute_matrices.pyx

```diff
@@ -43,6 +42,13 @@ class DistributeMatricesNode(bpy.types.Node, AnimationNode):
     meshMode = EnumProperty(name = "Mesh Mode", default = "VERTICES",
         items = meshModeItems, update = AnimationNode.refresh)
 
+    centerAlongX = BoolProperty(name = "Center Along X", default = True,
+        description = "Center the grid along the x axis.", update = AnimationNode.refresh)
```

Comment:
Instead of `AnimationNodes.refresh` the `executionCodeChanged` event should be used. 
The current one will trigger a rebuild of the sockets of the node which does seem to be necessary.